client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -147,6 +166,7 @@ const CalendarPage = () => {
           tasks={taskData}
           edittable={true}
           toggleTaskHandler={toggleTaskHandler}
+          createTaskReflection={createTaskReflection}
```

Comment:
For consistency, I would rename this to createTaskReflectionHandler

Reply:
Renamed.